test/inductor/test_flex_attention.py

```diff
@@ -1008,13 +1165,21 @@ def forward(self, L_args_0_: "f64[2, 2, 8, 4]", L_args_1_: "f64[2, 2, 8, 4]", L_
         l_args_1_ = L_args_1_
         l_args_2_ = L_args_2_
 
+        ones: "i32[1, 1, 1]" = torch.ones([1, 1, 1], dtype = torch.int32, device = device(type='cuda', index=0))
```

Comment:
This is starting to get quite ugly. I think we should figure out how to fix these in HOPs. cc: @zou3519 @ydwu4

Reply:
What exactly do you want fixed here?

Reply:
Today in the IR we end up printing out all of these dead nodes that aren't used anywhere.

Reply:
Removing the unused ```new_empty``` would be addressed as one of the follow-ups.